group_vars/all.yml

```diff
@@ -1240,8 +1240,15 @@ varnish_error_page_color_primary: "#444"
 # Max-age (in seconds) returned in Cache-Control header for HTML pages
 # (this header is rewritted by varnish as magento sets it unappropriately)
 varnish_html_cache_control_max_age: 60
-varnish_strip_google_gclid_params: yes
-
+varnish_strip_params:
+  - gclid
+  - utm_[^=]+
+  - gdftrk
+  - _ga
+  - mc_[^=]+
+  - trk_[^=]+
```

Comment:
Won't `[^=]` possibly eat too much, e.g. when `&` character is encountered?
I mean the case when query param has no value like `?param1=x&param2&param3=sth`.

Maybe it would be good to also exclude `&` character?